src/native/managed/cdacreader/tests/PrecodeStubsTests.cs

```diff
@@ -179,7 +179,8 @@ internal class PrecodeBuilder {
         public readonly MockMemorySpace.Builder Builder;
         public readonly MockMemorySpace.BumpAllocator PrecodeAllocator;
         public readonly MockMemorySpace.BumpAllocator StubDataPageAllocator;
-        public readonly Dictionary<DataType, Target.TypeInfo>? TypeInfoCache;
+
+        internal Dictionary<DataType, Target.TypeInfo>? Types { get; }
```

Comment:
Does this still need to be nullable? 

`Types = typeInfoCache ?? CreateTypeInfoCache(Builder.TargetTestHelpers);` seems like it doesn't want a `null` value. If `CreateTypeInfoCache` also returns a potential `null` value, that seems a bit odd.